packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-utils.tsx

```diff
@@ -0,0 +1,438 @@
+
+export const getPOIStatusMessages = (
+    status: typeof identity_status_codes[keyof typeof identity_status_codes],
+    auth_status?: TAuthStatus,
+    should_show_redirect_btn?: boolean,
+    is_from_external?: boolean
+) => {
+    const resubmitButton = (onClick?: () => void, platform_name?: string) => (
+        <React.Fragment>
+            <VerificationStatusActionButton
+                button_text={<Localize i18n_default_text={'Upload Document'} />}
+                onClick={onClick}
+            />
+            {!is_from_external && should_show_redirect_btn && (
+                <VerificationStatusActionButton
+                    button_text={<Localize i18n_default_text='Back to {{platform_name}}' values={{ platform_name }} />}
+                    onClick={onClick}
+                />
+            )}
+        </React.Fragment>
+    );
+
+    const verifiedButton = (onClick?: () => void, platform_name?: string) => {
+        if (is_from_external) return null;
+        if (auth_status?.needs_poa) {
+            return (
+                <VerificationStatusActionButton
+                    to={routes.proof_of_address}
+                    button_text={<Localize i18n_default_text='Submit proof of address' />}
+                />
+            );
+        }
+        if (should_show_redirect_btn)
+            return (
+                <VerificationStatusActionButton
+                    button_text={<Localize i18n_default_text='Back to {{platform_name}}' values={{ platform_name }} />}
+                    onClick={onClick}
+                />
+            );
+
+        return null;
+    };
+
+    const titles: Record<typeof status, React.ReactElement | string> = {
+        none: <Localize i18n_default_text={'Proof of identity verification not required'} />,
+        pending: <Localize i18n_default_text={''} />,
+        rejected: <Localize i18n_default_text={"You've reached the limit for uploading your documents."} />,
+        verified: <Localize i18n_default_text={'Your proof of identity is verified'} />,
+        expired: <Localize i18n_default_text={'New proof of identity document needed'} />,
+        suspected: <Localize i18n_default_text={"You've reached the limit for uploading your documents."} />,
+    };
+
+    const descriptions: Record<typeof status, React.ReactElement | null> = {
+        none: (
+            <Localize
+                i18n_default_text={
+                    'Your account does not need identity verification at this time.<0/>We will inform you if identity verification is required in the future.'
+                }
+                components={[<br key={0} />]}
+            />
+        ),
+        pending: <Localize i18n_default_text={''} />,
+        rejected: (
+            <Localize
+                i18n_default_text='Please contact us via <0>live chat</0>.'
+                components={[
+                    <span key={0} className='link link--orange' onClick={() => window.LC_API.open_chat_window()} />,
```

Comment:
Instead of span, can we use`SpanButton` component with `transparent`. This will remove Sonar cloud bugs like  - 
[Visible, non-interactive elements with click handlers must have at least one keyboard listener.](https://sonarcloud.io/project/issues?pullRequest=11420&open=AYvyD-fexzTLFybBUutC&id=binary-com_deriv-app)

Reply:
will try to add onKeyDown for resolving this error